xga/utils.py

```diff
@@ -660,10 +660,17 @@ def check_telescope_choices(telescope: Union[str, List[str]]) -> List[str]:
 ESASS_VERSION = None
 # This checks for an installation of eSASS
 eSASS_AVAIL = False
+which_evtool = shutil.which("evtool")
 if shutil.which("evtool") is None:
```

Comment:
Seeing as you defined `which_evtool` up above, you could replace the 'which' call in the if statement

Reply:
If you resolved this because you made alterations, I'm not sure you ever pushed the changes.

Reply:
I had resolved it because of local alterations yeah, I hadn't pushed yet because I've not worked with Pull Requests before so was not sure how to update the changes. I'll push those commits now.